pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -519,6 +516,8 @@ ssize_t sock_dtls_recv_aux(sock_dtls_t *sock, sock_dtls_session_t *remote,
     assert(data);
     assert(remote);
 
+    sock_udp_ep_t ep;
+    _session_to_ep(&remote->dtls_session, &ep);
```

Comment:
`ep` is labelled as `[out]` in the declaration of `sock_udp_recv_aux` so we can skip converting session to ep.